4_arm.ipynb

```diff
@@ -14,7 +16,23 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Motion generation using forward and inverse kinematics\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Objective</b>\n",
+    "    <p>In this notebook, we will learn how to control the arm of HSR using forward and inverse kinematics.</p>\n",
```

Comment:
"    <p>In this notebook, we will learn how to control the arm of the HSR using forward and inverse kinematics.</p>\n",